python/sdk/prelude_sdk/controllers/detect_controller.py

```diff
@@ -135,7 +135,7 @@ def download(self, test_id, filename):
             timeout=10
         )
         if res.status_code == 200:
-            return res.content
+            return res.text
```

Comment:
wonder if this should stay content - some attachments might be binary

Reply:
yes I updated it back after I pinged you yesterday!